pgcli/pgclirc

```diff
@@ -154,6 +154,12 @@ Token.Toolbar.Arg.Text = 'nobold'
 Token.Toolbar.Transaction.Valid = 'bg:#222222 #00ff5f bold'
 Token.Toolbar.Transaction.Failed = 'bg:#222222 #ff005f bold'
 
+# color of table
+# you can use token or custom colors 
+Token.Output.Header = "Token.Name.Tag"
+Token.Output.OddRow = ""
+Token.Output.EvenRow = "Token.Literal.String"
```

Comment:
I could see the use of other token names here a little confusing to the user. What about using HTML color codes or ANSI color names so it's very clear what colors these are?